src/packages/solid/components/Checkbox/Checkbox.tsx

```diff
@@ -36,15 +39,32 @@ const Checkbox: Component<CheckboxProps> = (props: CheckboxProps) => {
       {...props}
       tone={props.tone ?? styles.tone}
       style={[
+        ...[props.style].flat(),
         props.style?.Container,
         props.style?.Container?.[props.tone || styles.tone],
         styles.Container,
-        styles.Container[props.tone || styles.tone]
+        styles.Container?.[props.tone || styles.tone]
       ]}
       states={{
-        checked: props.checked
+        checked: props.checked,
+        tone: props.tone ?? styles.tone
```

Comment:
this would result in the tone being `tone` - we should directly assign either the props.tone or styles.tone